4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,225 +1,269 @@
+
+   > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+   
+   1. Open the `Client\appsettings.json` file.
+      1. Find the key `Domain` and replace the existing value with your Azure AD tenant name.
+      1. Find the key `TenantId` and replace the existing value with your Azure AD tenant ID.
+      1. Find the key `ClientId` and replace the existing value with the application ID (clientId) of `TodoListClient-aspnetcore-webapi` app copied from the Azure portal.
+      1. Find the key `ClientSecret` and replace the existing value with $pwdCredential.SecretText.
+      1. Find the key `TodoListScope` and replace the existing value with Scope.
+      1. Find the key `TodoListBaseAddress` and replace the existing value with Web.
+            
+  **For more information, visit** [Register Application AAD](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+
+  </details>
+
+**OR**
+
+- use PowerShell scripts that:
+  - **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you.
+  - modify the projects' configuration files.  
+  <details>
+   <summary>Expand this section if you want to use this automation:</summary>
+      
+   > :warning: If you have never used **Azure AD Powershell** before, we recommend you go through the [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md) once to ensure that your environment is prepared correctly for this step.
+    
+    1. On Windows, run PowerShell as **Administrator** and navigate to the root of the cloned directory
+    1. If you have never used Azure AD Powershell before, we recommend you go through the [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md) once to ensure that your environment is prepared correctly for this step.
+    1. In PowerShell run:
+    
+       ```PowerShell
+       Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+       ```
+    
+    1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+    1. For interactive process - in PowerShell run:
+    
+       ```PowerShell
+       cd .\AppCreationScripts\
+       .\Configure.ps1 -TenantId "Optional - your tenant id" -Environment "Optional - Azure environment, defaults to 'Global'"
+       ```
+    
+       > Other ways of running the scripts are described in [App Creation Scripts](./AppCreationScripts/AppCreationScripts.md)
+       > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+
+  </details>
+   
+
+### Variation: web app using client certificates
+
+Follow [README-use-keyvault-certificate.md](README-use-keyvault-certificate.md) to know how to use this option.
+### Step 4: Running the sample
+
+    
+> For Visual Studio users
+>
+> Clean the solution, rebuild the solution, and run it.
```

Comment:
remove this, lets just stick with cmdlines

Reply:
@kalyankrishna1 
What exactly to remove here?